### ISSUE TITLE
customize the 'scatterplot'

### ISSUE THREAD

**Issue by demis001** [GMOD/jbrowse#1462]
How to customize the scatterplot for bigwig? The dots is very tiny with defult setting. Anyone who guide me please. I can also edit the node source code if you guide me where to find the style file for this module. 

Best,
@demis001

**Comment by cmdcolin** [comment]
The scatterplot for bigwig, aka the "No fill" option, I agree could use improvement


Look for noFill in here

https://github.com/GMOD/jbrowse/blob/master/src/JBrowse/View/Track/Wiggle/XYPlot.js

To get started with modifying jbrowse source code run these steps basically

```
git clone https://jbrowse.org/gmod/jbrowse
cd jbrowse
yarn
yarn watch
```

This will build jbrowse and keep it in "watch mode" for changes

Then in another window you can run yarn start to start a server, or if you did git clone into an apache web folder or similar, then you can just use the apache web server path

**Comment by cmdcolin** [comment]
Also, if you have context for this e.g. I have seen things like copy number variant data comes in various forms instead of bigwig, we could make a data adapter for those formats instead of bigwig. There is for example https://github.com/elsiklab/gwasviewer which is kind of a scatter plot and allows for mouseover and click actions on the features

**Comment by demis001** [comment]
@cmdcolin 

Thanks! I have looked the gwasviewer plugin and installed it. I have installed other plugin too. But, It didn't work for some reason. Nothing shown up.  I am trying to show methylation data. The last column is a methylation ratio similar to p-value in mutation call. I converted frorm wig to bed file and gz it based on the readme file. Any idea? 

I have a bed file similar to this one shown below, the last column is a methylation ratio. I added the *.bed.gz file to the webroot and added the following conf. But, it didn't work, I have installed the other plugin including this one. Any idea?

{
"maxFeatureScreenDensity": 6,
"storeClass" : "JBrowse/Store/SeqFeature/BEDTabix",
"urlTemplate" : "../../rawData/data.bed.gz",
"label" : "brain_GCmap_vPlotter",
"type" : "GWASViewer/View/Track/VariantPlotter"
}

chr1 147971108 147971158 id-1 0.590000
chr1 147971146 147971196 id-2 0.120000
chr1 147971184 147971234 id-3 0.110000
chr1 147971222 147971272 id-4 0.760000
